Utils/python/pfnInPath.py

```diff
@@ -0,0 +1,18 @@
+# from https://github.com/cms-sw/cmssw/blob/master/FWCore/ParameterSet/python/pfnInPath.py
```

Comment:
Why does this need to be included in TreeMaker? Why can't we import the CMSSW version?

Reply:
The CMSSW version is not available in 10_2_X.

Reply:
Ah, okay. I must have only checked 10_6_X. This file should be removed whenever these changes get forward ported to the UL branch.